tests/album-read.test.js

```diff
@@ -0,0 +1,76 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const db = require('../src/db');
+const app = require('../src/app');
+
+describe('Read Albums', () => {
+    let artists;
+    let albums;
+    beforeEach(async () => {
+        const artistResponses = await Promise.all ([
+            db.query(
+                'INSERT INTO Artists (name, genre) VALUES ($1, $2) RETURNING *', 
+                ['Taylor Swift','Pop']
+            ),
+            db.query(
+                'INSERT INTO Artists (name, genre) VALUES ($1, $2) RETURNING *',
+                ['Typhoons', 'Rock']
+            ),
+            db.query(
+                'INSERT INTO Artists (name, genre) VALUES( $1, $2) RETURNING *',
+                ['Tom Player','Instrumental']
+            ),
+        ]);
+        artists = artistResponses.map(({ rows }) => rows[0]);
+
+
+        const albumResponses = await Promise.all([
```

Comment:
Extra clear lines here, you never need more than one. Rather than me pointing out every instance it would be good to spend a few minutes checking over the files and editing them to sort these instances out when you have a minute :)